definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -1532,6 +1532,20 @@
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "CRU"
   },
+  {
```

Comment:
I may be being silly, but are these two new entries not exactly the same? We just need one right?